2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*sysRole).GetRoleList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_role.go:71
3.  gfast/app/system/api.(*user).GetInfo
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_user.go:112
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*sysRole).GetRoleList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_role.go:83
3.  gfast/app/system/api.(*user).GetInfo
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_user.go:112
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*sysRole).GetRoleList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_role.go:84
3.  gfast/app/system/api.(*user).GetInfo
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_user.go:112
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*sysRole).GetRoleList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_role.go:71
3.  gfast/app/system/api.(*user).GetRouters
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_user.go:163
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*sysRole).GetRoleList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_role.go:83
3.  gfast/app/system/api.(*user).GetRouters
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_user.go:163
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*sysRole).GetRoleList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_role.go:84
3.  gfast/app/system/api.(*user).GetRouters
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_user.go:163
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/service.(*rule).GetIsMenuStatusList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:19
4.  gfast/app/system/service.(*sysUser).GetAllMenus
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_user.go:203
5.  gfast/app/system/api.(*user).GetRouters
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_user.go:181
6.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
7.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
8.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
9.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
10. gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/service.(*rule).GetIsMenuStatusList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:19
4.  gfast/app/system/service.(*sysUser).GetAllMenus
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_user.go:203
5.  gfast/app/system/api.(*user).GetRouters
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_user.go:181
6.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
7.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
8.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
9.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
10. gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:39 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:47 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.sysDictData.GetDictWithDataByType
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_dict_data.go:69
3.  gfast/app/system/api.(*dictData).GetDictData
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_dict_data.go:114
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:47 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:47 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.sysDictData.GetDictWithDataByType
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_dict_data.go:96
3.  gfast/app/system/api.(*dictData).GetDictData
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_dict_data.go:114
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:47 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:47 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:47 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:49 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.sysDictData.GetDictWithDataByType
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_dict_data.go:69
3.  gfast/app/system/api.(*dictData).GetDictData
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_dict_data.go:114
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:49 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:49 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.sysDictData.GetDictWithDataByType
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_dict_data.go:96
3.  gfast/app/system/api.(*dictData).GetDictData
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_dict_data.go:114
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:49 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:49 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:49 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:51 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*sysWebSet).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_web_set.go:25
3.  gfast/app/system/api.(*sysWebSet).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_web_set.go:40
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:51 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*sysWebSet).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_web_set.go:34
3.  gfast/app/system/api.(*sysWebSet).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_web_set.go:40
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 16:59:51 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 16:59:51 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:09 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/common/service.(*sysConfig).GetConfigByKey
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/sys_config.go:129
3.  gfast/app/common/adapter.UploadLocalAdapter.getUpConfig
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:192
4.  gfast/app/common/adapter.UploadLocalAdapter.upByType
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:64
5.  gfast/app/common/adapter.UploadLocalAdapter.UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:28
6.  gfast/app/common/adapter.upload.UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload.go:63
7.  gfast/app/system/api.(*upload).UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/upload.go:28
8.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
9.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
10. gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
11. github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
12. gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:00:09 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/common/service.(*sysConfig).GetConfigByKey
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/sys_config.go:138
3.  gfast/app/common/adapter.UploadLocalAdapter.getUpConfig
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:192
4.  gfast/app/common/adapter.UploadLocalAdapter.upByType
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:64
5.  gfast/app/common/adapter.UploadLocalAdapter.UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:28
6.  gfast/app/common/adapter.upload.UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload.go:63
7.  gfast/app/system/api.(*upload).UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/upload.go:28
8.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
9.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
10. gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
11. github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
12. gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:00:09 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/common/service.(*sysConfig).GetConfigByKey
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/sys_config.go:129
3.  gfast/app/common/adapter.UploadLocalAdapter.getUpConfig
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:192
4.  gfast/app/common/adapter.UploadLocalAdapter.upByType
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:76
5.  gfast/app/common/adapter.UploadLocalAdapter.UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:28
6.  gfast/app/common/adapter.upload.UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload.go:63
7.  gfast/app/system/api.(*upload).UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/upload.go:28
8.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
9.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
10. gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
11. github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
12. gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:00:09 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/common/service.(*sysConfig).GetConfigByKey
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/sys_config.go:138
3.  gfast/app/common/adapter.UploadLocalAdapter.getUpConfig
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:192
4.  gfast/app/common/adapter.UploadLocalAdapter.upByType
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:76
5.  gfast/app/common/adapter.UploadLocalAdapter.UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload_local.go:28
6.  gfast/app/common/adapter.upload.UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/common/adapter/upload.go:63
7.  gfast/app/system/api.(*upload).UpImg
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/upload.go:28
8.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
9.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
10. gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
11. github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
12. gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:00:09 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:09 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:13 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:13 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:18 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.sysDictData.GetDictWithDataByType
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_dict_data.go:69
3.  gfast/app/system/api.(*dictData).GetDictData
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_dict_data.go:114
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:00:18 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:18 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:18 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.sysDictData.GetDictWithDataByType
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_dict_data.go:96
3.  gfast/app/system/api.(*dictData).GetDictData
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_dict_data.go:114
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:00:18 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:18 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:21 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*sysWebSet).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_web_set.go:25
3.  gfast/app/system/api.(*sysWebSet).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_web_set.go:40
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:00:21 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*sysWebSet).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_web_set.go:34
3.  gfast/app/system/api.(*sysWebSet).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_web_set.go:40
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:00:21 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:00:21 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:16:52 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.sysDictData.GetDictWithDataByType
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_dict_data.go:69
3.  gfast/app/system/api.(*dictData).GetDictData
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_dict_data.go:114
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:16:52 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:16:52 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.sysDictData.GetDictWithDataByType
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_dict_data.go:96
3.  gfast/app/system/api.(*dictData).GetDictData
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_dict_data.go:114
4.  gfast/middleware.Auth
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:72
5.  gfast/middleware.Ctx
    /Users/apple/work/code/go/gfastv2/gfast/middleware/auth.go:46
6.  gfast/app/system/api.(*auth).authAfterFunc
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:126
7.  github.com/goflyfox/gtoken/gtoken.(*GfToken).authMiddleware
    /Users/apple/go/pkg/mod/github.com/goflyfox/gtoken@v1.4.3/gtoken/gtoken.go:324
8.  gfast/middleware.CORS
    /Users/apple/work/code/go/gfastv2/gfast/middleware/cors.go:15
 
2021-09-12 17:16:52 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Get
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:125
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:36
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:16:52 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:16:52 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89: dial tcp 127.0.0.1:6379: connect: connection refused 
Stack:
1.  gfast/app/common/service.(*cacheTagService).Set
    /Users/apple/work/code/go/gfastv2/gfast/app/common/service/cache.go:89
2.  gfast/app/system/service.(*rule).GetMenuList
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_auth_rule.go:47
3.  gfast/app/system/api.(*sysOperLog).OperationLog
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/sys_oper_log.go:86
 
2021-09-12 17:30:05 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_user_online.go:61: dial tcp [::1]:3306: connect: connection refused, SELECT COUNT(1) FROM `sys_user_online`
Stack:
1.  gfast/app/system/service.(*online).GetOnlineListPage
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_user_online.go:61
2.  gfast/app/system/api.(*auth).CheckUserOnline
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:165
 
2021-09-12 17:30:05 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:167: 获取总行数失败 
Stack:
1.  gfast/app/system/api.(*auth).CheckUserOnline
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:167
 
2021-09-12 17:40:05 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_user_online.go:61: dial tcp [::1]:3306: connect: connection refused, SELECT COUNT(1) FROM `sys_user_online`
Stack:
1.  gfast/app/system/service.(*online).GetOnlineListPage
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_user_online.go:61
2.  gfast/app/system/api.(*auth).CheckUserOnline
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:165
 
2021-09-12 17:40:05 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:167: 获取总行数失败 
Stack:
1.  gfast/app/system/api.(*auth).CheckUserOnline
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:167
 
2021-09-12 18:40:05 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_user_online.go:61: dial tcp [::1]:3306: connect: connection refused, SELECT COUNT(1) FROM `sys_user_online`
Stack:
1.  gfast/app/system/service.(*online).GetOnlineListPage
    /Users/apple/work/code/go/gfastv2/gfast/app/system/service/sys_user_online.go:61
2.  gfast/app/system/api.(*auth).CheckUserOnline
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:165
 
2021-09-12 18:40:05 [ERRO] /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:167: 获取总行数失败 
Stack:
1.  gfast/app/system/api.(*auth).CheckUserOnline
    /Users/apple/work/code/go/gfastv2/gfast/app/system/api/auth.go:167
 
